client/client.go

```diff
@@ -143,7 +143,33 @@ func (c *Client) getFile(docID int) error {
 func (c *Client) SearchWord(word string) ([]string, float64, error) {
 	possibleDocs := c.server.SearchWord(c.indexer.ComputeTrapdoors(word))
 	args := make([]string, len(possibleDocs)+2)
-	args[0] = "-lZ"
+	args[0] = "-lZw"
```

Comment:
Everything from here down is copied code -- you should refactor into a helper function that takes `possibleDocs`.